### PR TITLE
Migrate openconfig-system to transformer

### PR DESCRIPTION
**Pull request by joyas-joseph**

### CODE REVIEW

File: src/translib/transformer/xfmr_system.go [hunk 1]
```diff
@@ -0,0 +1,302 @@
...
+    syscall.Flock(int(jsonFile.Fd()),syscall.LOCK_EX)
+    log.Infof("syscall.Flock done")
+
+    defer jsonFile.Close()
+    defer log.Infof("jsonFile.Close called")
+    defer syscall.Flock(int(jsonFile.Fd()), syscall.LOCK_UN);
+    defer log.Infof("syscall.Flock unlock  called")
+
+    byteValue, _ := ioutil.ReadAll(jsonFile)
+    json.Unmarshal(byteValue, &jsonsystem)
```
Comment (kwangsuk) [inline src/translib/transformer/xfmr_system.go]: Can you please check a parse error?
err := json.Unmarshal(byteValue, &jsonsystem)
if err != nil {
    . . .
}

File: config/transformer/models_list [hunk 2]
```diff
@@ -5,3 +5,5 @@ openconfig-acl-annot.yang
 openconfig-network-instance.yang
 openconfig-network-instance-annot.yang
 sonic-vxlan.yang
```
Comment (kwangsuk) [inline config/transformer/models_list]: This is not part of your code. But, can you please remove the sonic-vxlan.yang for others? The models_list is intended for only openconfig modules. thanks

File: src/translib/sys_app.go [hunk 3]
```diff
@@ -45,6 +45,7 @@ type SysApp struct {
 
 func init() {
     log.Info("SysApp: Init called for System module")
+	return
```
Comment (kwangsuk) [inline src/translib/sys_app.go]: can we remove the sys_app.go since your pull request replaces it?

File: config/transformer/models_list (hunk 2 above)
Comment (joyas-joseph) [reply config/transformer/models_list]: Done

File: src/translib/sys_app.go (hunk 3 above)
Comment (joyas-joseph) [reply src/translib/sys_app.go]: Done

**Review by kwangsuk — APPROVED** [review]
The change looks good.

**Pull request merged**